tests/v2/validation/provisioning/rke2/custom_cluster_test.go

```diff
@@ -47,6 +49,10 @@ func (c *CustomClusterProvisioningTestSuite) SetupSuite() {
 
 	c.client = client
 
+	c.kubernetesVersions, err = kubernetesversions.Default(c.T(), c.client, clusters.RKE2ClusterType.String(), c.kubernetesVersions)
+	require.NoError(c.T(), err)
+
+
```

Comment:
last nit I promise 😄 there's an extra space here that isn't seen in rke1 or k3s.

Reply:
pending this, everything else lgtm

Reply:
resolved!  no more extra space 😁